package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "witnet-ethereum-bridge",
-  "version": "0.4.3",
+  "version": "0.5.0",
```

Comment:
As a rule of thumb, version bumps should go into their own commit at the end of the PR, clearly stating `chore: bump package version to 0.5.0`